daemons/dss-index/app.py

```diff
@@ -1,4 +1,6 @@
-import os, sys, json, boto3, domovoi
+import os, sys, json, boto3, logging
```

Comment:
Cleaned this up into one import per line. Broke out `boto3` into the third-party group.